### PR TITLE
Ensure stdlib imports are recognized

### PR DESCRIPTION
**Pull request by ulope**
This PR adds a test that ensures stdlib imports are recognized as such.
It also fixes `concurrent` not being recognized as a stdlib import.

### CODE REVIEW

**Comment by pgjones** [comment]
Thanks, I will get to this, however the bug report is taking a little effort to fix.

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        ast.parse(
+            code,
+            filename
+        ),
+        str(filename)
+    )
+
+
+@pytest.fixture
+def flake8_linter():
```
Comment (pgjones) [inline tests/conftest.py]: @ulope I'm not sure why this function needs to be a fixture, why not just call the function?

Comment (sigmavirus24) [inline tests/conftest.py]: I'm a little confused as well. There doesn't seem to be any inherent value in it being a fixture other than not having to import it to use it below.

Comment (ulope) [inline tests/conftest.py]: You're right as it's used now there is no real benefit to it being a fixture. I had initially planned to do a bit more refactoring and use this fixture in all the places where a linter is used. I abandoned this however as I felt it was out of scope for this PR but then didn't think to revert this to a plain function.

If you want I can update the PR to use a plain function.

Comment (pgjones) [inline tests/conftest.py]: Yes please.

**Comment by ulope** [comment]
Updated. The helper is a regular function now.

**Pull request merged**